questions.md

```diff
@@ -461,5 +461,9 @@ in your own life or been asked by someone?
 
 Please share!
 
+<a href="https://twitter.com/home?status=https%3A//github.com/dwyl/start-here/blob/master/questions.md">Share on Twitter</a><br>
+<a href="https://www.facebook.com/sharer/sharer.php?u=https%3A//github.com/dwyl/start-here/blob/master/questions.md">Share on Facebook</a><br>
```

Comment:
**`while`** I feel that Facebook is a _fundamentally_ [evil](https://github.com/dwyl/learn-react/issues/23) (_to the **core**_) company intent on harvesting people's personal information, data and intimate thoughts/photos/feelings, to then enable _hyper_ targeted and _manipulative_ advertising (_which has caused real [**harm**](https://childmind.org/article/how-using-social-media-affects-teenagers/) or even [**death**](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3477910) to many people_) ...
I _acknowledge_ that Facebook is still used by people to _communicate_ with their "friends" and loved ones. 

On that basis, and until a viable alternative is available, I am (_reluctantly_) "OK" with including this link.